src/main/java/jmgomez/apipolicy/feignClient/PolicyClient.java

```diff
@@ -1,24 +1,26 @@
 package jmgomez.apipolicy.feignClient;
 
+import jmgomez.apipolicy.config.FeignClientConfig;
 import jmgomez.apipolicy.model.Accident;
 import jmgomez.apipolicy.model.dto.PolicyDto;
 import jmgomez.apipolicy.model.dto.PolicyDtoCov;
 import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 
-@FeignClient(name = "policyClient", url = "http://localhost:8081")
+@FeignClient(name = "policyClient", url = "http://localhost:8081", configuration = FeignClientConfig.class)
 public interface PolicyClient {
 
-    @GetMapping(value = "/polizas?dni={id}", headers = "Accept-application/json")
+    @GetMapping(value = "/polizas?dni={id}", headers = "Accept-application/json", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Do we really need this header and consumes configuration? Headers one doesn't even seem to be correct.